OpenIPSL/Data/PowerPlant/HowTo.mo

```diff
@@ -0,0 +1,13 @@
+within OpenIPSL.Data.PowerPlant;
+model HowTo
+  extends Modelica.Icons.Information;
+  annotation (DocumentationClass=true, Documentation(info="<html>
+<h4>Data Sets Usage Guide</h4>
+<p>The following short guide is intended to provide a brief description on the use of the <strong>parameter data sets for generation units</strong> contained in this package.</p>
+<p><strong>Example 1</strong> of the <strong>OpenIPSL Library Tutorial</strong> explains the process required to create a simple network model along with a <strong>generation group</strong> sub-model. Although it is not necessary to create a model only for the generation group, it is highly recommended as it promotes modularity and flexibility. It should be noted that the network model can grow in size and complexity, as the IEEE14 and N44 examples demonstrate. For these last two cases, there is a generator group model for each generator connected to the network.</p>
+<p>To be able to make use of the parameterized records in this package, simply <strong><span style=\"color: #00007f;\">drag and drop an instance of the GU_Dynamics record</span></strong> onto the interest generation group model.</p>
+<p>Then, propagate the record to the upper level and double click on the generator component. Go to the dropdown list of the propagated <strong>GUnitDynamics</strong> and <strong><span style=\"color: #00007f;\">choose the set of parameters that best suits your needs</span></strong>. The available parameters can be inspected later by first clicking on the button to the right of the drop-down list, and then on the button of the corresponding generator set component (i.e. machine, excitation system, etc.).</p>
+<p>The last step is to double-click on each of the model components and <strong><span style=\"color: #00007f;\">insert references to match the parameters with their corresponding values from the GU_Dynamics record</span></strong>. For example, after double-clicking on the machine component, its inertia parameter H can be assigned by reference using the expression <em>gU_Dynamics.guDynamics.machine.H</em>. The reference can be typed, copied and pasted or can be explored using the Insert Component Reference functionality.</p>
```

Comment:
Which code expression are you referring to?

`<em>gU_Dynamics.guDynamics.machine.H</em>`  =>  `<code>gU_Dynamics.guDynamics.machine.H</code>` ?

I don't really see that expression as a real code expression because it does not have any reserved keyword or a complete Modelica-based syntax. Please correct me if I am wrong...

Reply:
Yes, that is what I meant. It is easier to identify class names when they are highlighted as code.

Reply:
I can not see that this has been changed to use `<code>`. Could you please fix this.

Reply:
Alright! I think it is fixed now. See [commit 0dc9409](https://github.com/OpenIPSL/OpenIPSL/pull/329/commits/0dc94098933ae5cb06852b1a5cfddb7752ff854b)

Reply:
Not really. The How-To is really confusing and not 100% clear what you are referring to. The use of colours and bold fonts makes it even harder to follow. To make things simpler. Start with removing **all** font modifications. Then add href links to classes that users should be using. Here you will also find that for example there is no `Example 1` but only a `Example_1` class which if you actually link to it is clear. The same goes for ´GUDynamics`  record. There is a package and a record and linking to the correct one again is clearer plus no need for additional font noise since the link will stand out on its own.

Reply:
Still some issues:
- The `Example_1` is still called `Example 1` in the How-To
- You might want to link "examples" in the last line to the `Examples` package instead of bold-fonting it.

Reply:
@maguilerac It looks like you are linking `GUDynamics` in the HowTo to the GUDynamics **package** instead of the record. Now the HowTo says "
Drag and drop an instance of the GUDynamics record on the diagram layer of the model." So I assume it really should be linked to the record.